src/solver/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -276,23 +276,6 @@ bool PerformanceConfigConvOclBwdWrw2<N_BATCH_LOOPS>::IsValid(const ConvolutionCo
         return false;
     }
 
-    // For group config, reducing search space
```

Comment:
Dead code?

Reply:
This code doesn't do anything. The condition `(n_output_channels_per_group % 1 == 0 && n_out_channels_per_tile % 1 == 0)` is always true because `x % 1` is always 0. This makes the early return never reached. Beyond this early return, there are no other side effects, so all this can be removed.

Please double check that these assumptions are correct.

Reply:
Kill it